### ISSUE TITLE
Mass Farming Bug

### ISSUE THREAD

**Issue by tscheer100** [Firoso/Valheim-Dusk-Niflheim#12]
Mas farming is behaving in an unexpected manner.
Possibly an indestructible items bug.

**Comment by PolyphonyRequiem** [comment]
Yup, indestructible items bug.  Not sure how to address this at this time.
